### PR TITLE
feat(translation): Add Traditional Chinese translation

### PR DESCRIPTION
**Pull request by patw0929**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 7  |
| Bug fix?         | no        |
| New feature?     | no        |
| Includes tests?  | no        |
| All Tests pass?  | yes       |
| Related ticket?  | #439        |

---

## Description

* Added Traditional Chinese translation
* Replaced a string `at` with i18n translate function in `events.screen.js`

I understand Traditional Chinese was not mentioned in #439, but I still spent some time to translate strings to Traditional Chinese.

Please provide suggestions if you are Traditional Chinese speaker.
Many thanks! 😄😄😄

### CODE REVIEW

File: src/locale/languages/zh-tw.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+  auth: {
+    login: {
+      connectingToGitHub: '連線至 GitHub...',
+      preparingGitPoint: 'GitPoint 準備中...',
+      cancel: '取消',
+      welcomeTitle: '歡迎使用 GitPoint',
+      welcomeMessage: '完全免費，功能最強大的 GitHub 應用程式！',
+      notificationsTitle: '通知設定',
+      notificationsMessage: '檢視並設定所有未讀與參與的通知',
+      reposTitle: '儲存庫與使用者',
```
Comment (ArvinH) [inline src/locale/languages/zh-tw.js]: @patw0929 Thanks for your translating! It looks all good except that I'm not sure if we call `repository` as "儲存庫“, more like "版本庫" or just use repository directly. What do you think?

Comment (patw0929) [reply src/locale/languages/zh-tw.js]: Thank you for your advice! 👍  I'll choose `版本庫` here.

**Comment by chinesedfan** [comment]
👍 LGTM. For traditional Chinese, people in Taiwan and Hong Kong are usually more familiar than Mainland China.

**Comment by housseindjirdeh** [comment]
Beautiful 😍

**Pull request merged**